packages/typespec-go/src/tcgcadapter/types.ts

```diff
@@ -507,7 +509,7 @@ export class typeAdapter {
       usage |= go.UsageFlags.Output;
     }
 
-    const annotations = new go.ModelAnnotations(false, model.isFormDataType);
+    const annotations = new go.ModelAnnotations(false, (model.usage & tcgc.UsageFlags.MultipartFormData) === tcgc.UsageFlags.MultipartFormData);
```

Comment:
This was a breaking change, `isformDataType` was removed.

Reply:
Why was this removed? It was much easier to deal with how it was before.